pkg/cmd/server/api/v1/types_test.go

```diff
@@ -294,9 +299,12 @@ func TestMasterConfig(t *testing.T) {
 				{Provider: runtime.EmbeddedObject{Object: &internal.OpenIDIdentityProvider{}}},
 			},
 			SessionConfig: &internal.SessionConfig{},
+			Templates:     &internal.OAuthTemplates{},
+		},
+		AssetConfig: &internal.AssetConfig{
```

Comment:
Yeah, that was the UI extension configuration, I wanted the substructs exposed in the unit test